packages/telemetry/src/client.ts

```diff
@@ -218,7 +218,6 @@ export class TelemetryClient implements ITelemetryClient {
       if (this.enabled) {
         this.backend.sendEvent(event);
         this.session.touch();
-        this.backend.flush();
```

Comment:
@dividedmind seems like a focused but significant behaviour change -- maybe is innnocent and even useful, but I still wonder -- is it worth making flush/noflush a configurable option?

Typically there are tradeoffs between buffering and data persistence. Few questions arise, and answers are not immediately obvious without full code inspection:
1) does the change imply that system will be *always* buffering data? 
2) If so, does it match user expectations and documentation? Is flush guaranteed eventually?
3) Will there be arisk of losing some telemetry (buffered but not flushed), is it acceptable?

I understand that answers could be trivial but I think questions are still worth asking.
Would very much appreciate if you'll find time to reflect on them. Thanks!